README.md

```
@@ -48,133 +48,100 @@
-2. [Frontend](https://github.com/AOSSIE-Org/TODO/tree/main/frontend) (if separate)
-3. [Backend](https://github.com/AOSSIE-Org/TODO/tree/main/backend) (if separate)
+[Main Repository](https://github.com/AOSSIE-Org/OrgExplorer)
+
 
 ---
 
 ## 🏗️ Architecture Diagram
 
 TODO: Add your system architecture diagram here
-
 ```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)